ci-operator/step-registry/ipi/conf/azurestack/ipi-conf-azurestack-commands.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+REGION="ppe3"
+echo "Azure region: ${REGION}"
+
+workers=3
+if [[ "${SIZE_VARIANT}" == "compact" ]]; then
+  workers=0
+fi
+master_type=null
+if [[ "${SIZE_VARIANT}" == "xlarge" ]]; then
+  master_type=Standard_D32_v3
+elif [[ "${SIZE_VARIANT}" == "large" ]]; then
+  master_type=Standard_D16_v3
+elif [[ "${SIZE_VARIANT}" == "compact" ]]; then
+  master_type=Standard_D8_v3
+fi
+echo $master_type
+
+ENDPOINT="https://management.ppe3.stackpoc.com"
+echo "ASH ARM Endpoint: ${ENDPOINT}"
+cp "/var/run/azurestack-cluster-secrets/service-principal" "${SHARED_DIR}/osServicePrincipal.json"
+
+cat >> "${CONFIG}" << EOF
+baseDomain: ppe.upi.azurestack.devcluster.openshift.com
+credentialsMode: Manual
+metadata:
+  name: ci-op
```

Comment:
Tests are failing because there are conflicts with creating hosted zones with the same names. 

The immediate fix is to make the cluster name (metadata.name) unique. We can use the pattern here: https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/ipi-conf-commands.sh

Subsequently, we can look into reusing a DNS zone and just creating records in the existing zone.